googlecloudstorage/googlecloudstorage.go

```diff
@@ -181,11 +207,34 @@ func parsePath(path string) (bucket, directory string, err error) {
 	return
 }
 
+func getServiceAccountClient(keyJsonfilePath string) (*http.Client, error) {
+	data, err := ioutil.ReadFile(os.ExpandEnv(keyJsonfilePath))
+	if err != nil {
+		return nil, fmt.Errorf("error opening credentials file: %v", err)
+	}
+	conf, err := google.JWTConfigFromJSON(data, storageConfig.Scopes...)
+	if err != nil {
+		return nil, fmt.Errorf("error processing credentials: %v", err)
+	}
+	return conf.Client(context.TODO()), nil
+}
+
 // NewFs contstructs an Fs from the path, bucket:path
 func NewFs(name, root string) (fs.Fs, error) {
-	oAuthClient, err := oauthutil.NewClient(name, storageConfig)
-	if err != nil {
-		log.Fatalf("Failed to configure Google Cloud Storage: %v", err)
+	var oAuthClient *http.Client
+	var err error
+
+	serviceAccountPath, _ := fs.ConfigFile.GetValue(name, "service_account_file")
```

Comment:
Oh cool, usually when I see `MustValue` I assume the code would `panic` otherwise. Maybe worth renaming to `ValueWithDefault`? :)